composer.json

```diff
@@ -36,5 +36,6 @@
         "branch-alias": {
             "dev-master": "1.0-dev"
         }
-    }
+    },
+    "abandoned": "league/commonmark"
```

Comment:
This should be removed.

Reply:
It's a valid schema property: https://getcomposer.org/doc/04-schema.md#abandoned

Reply:
Which is not read by Packagist...

Reply:
It only exists so that lock files can encode abandonment. It is not supposed to be used in the way you are using it.

Reply:
No, but it is by composer. That's why you see those warnings in the CLI that show you packages have been abandoned. This change isn't about packagist, you're making it about it.

Reply:
No? Composer only shows you a warning if Packagist has the package recorded as abandoned (unless you explicitly load the repo as a "repository").

Reply:
Idk why you're fighting this. This is how all the other CommonMark extensions have been migrated. I'm just following up the last little bit so these projects point to the correct repo that replaced them (which is how this schema property actually is intended to be used). Clearly there's going to be some additional static metadata that will need to be set on packagist by the original author, I'm not disputing that; that's a separate issue. This is about making sure the composer.json reflects the current state of the project. I'm not changing this.